src/geophires_x/WellBores.py

```diff
@@ -88,15 +164,15 @@ def WellPressureDrop(model: Model, Taverage: float, wellflowrate: float, welldia
     rhowater = np.array([
         density_water_kg_per_m3(
             t,
-            pressure=model.reserv.lithostatic_pressure(),
+            pressure=model.reserv.lithostatic_pressure(model.reserv.rhorock.value, model.reserv.depth.value),
         )
         for t in Taverage
     ])  # replace with correlation based on Tprodaverage
 
     muwater = np.array([
         viscosity_water_Pa_sec(
             t,
-            pressure=model.reserv.lithostatic_pressure(),
+            pressure=model.reserv.lithostatic_pressure(model.reserv.rhorock.value, model.reserv.depth.value),
```

Comment:
Earlier comment re: `model.reserv.lithostatic_pressure()` should not take parameters - applies here and everywhere else the same change has been made (that I've seen in the PR thus far)